tests/Unit/ItemTest.php

```diff
@@ -3262,6 +3262,50 @@ public static function getPermalinkDataProvider(): array
                 ,
                 'http://example.com/',
             ],
+            'Test RSS 2.0 with channel link and enclosure from another domain' => [
+<<<XML
+<rss version="2.0" xmlns:media="http://search.yahoo.com/mrss/">
+    <channel>
+		<link>http://example.net/tests/</link>
+        <item>
+			<link>/tests/1/</link>
+			<media:content url="http://example.com/images/1.jpg" medium="image"></media:content>
+		</item>
+    </channel>
+</rss>
+XML
+                ,
+                'http://example.net/tests/1/',
```

Comment:
Was wrongly returning `http://example.com/tests/1/` before this patch (side-effect of the enclosure)